benchmarks/profile_conversion_time.py

```diff
@@ -0,0 +1,101 @@
+# coding: utf-8
+"""
+Profiles the conversion of a Keras model.
+"""
+import fire
+import tensorflow as tf
+from tf2onnx import tfonnx
+from tensorflow.keras.applications import MobileNet
```

Comment:
Iffy to depend on tf.keras since we must support older tf-1.x versions.

Reply:
We can remove the script from CI, no issue with that, it takes 30 seconds. My plan was to only run it with the latest version of tensorflow. However, the script is failing and it was working a month ago. So I was wondering what change could have caused that. Either my script is wrong, either there is something not covered by the unit tests...

Reply:
can we not start using fire ? We don't use it anywhere else and want to limit the dependencies.

Reply:
Dependency removed.